documentation/README.md

```diff
@@ -30,6 +30,217 @@
 - [explode](#explode)
 - [extend](#extend)
 - [filter](#filter)
+- [flatMap](#flatmap)
+- [flip](#flip)
+- [fold](#fold)
+- [foldRight](#foldright)
+- [forEach](#foreach)
+- [greaterOrEqual](#greaterorequal)
+- [greaterThan](#greaterthan)
+- [hasOwnProperty](#hasownproperty)
+- [head](#head)
+- [identity](#identity)
+- [ifThen](#ifthen)
+- [ifThenElse](#ifthenelse)
+- [implode](#implode)
+- [inc](#inc)
+- [isBetween](#isbetween)
+- [isBoolean](#isboolean)
+- [isFalse](#isfalse)
+- [isFalsy](#isfalsy)
+- [isFunction](#isfunction)
+- [isNull](#isnull)
+- [isNumber](#isnumber)
+- [isObject](#isobject)
+- [isPlainObject](#isplainobject)
+- [isString](#isstring)
+- [isTrue](#istrue)
+- [isTruthy](#istruthy)
+- [isTypeOf](#istypeof)
+- [isUndefined](#isundefined)
+- [isUnknown](#isunknown)
+- [join](#join)
+- [keys](#keys)
+- [last](#last)
+- [length](#length)
+- [lessOrEqual](#lessorequal)
+- [lessThan](#lessthan)
+- [looseEqual](#looseequal)
+- [map](#map)
+- [match](#match)
+- [max](#max)
+- [method](#method)
+- [min](#min)
+- [minus](#minus)
+- [nand](#nand)
+- [noop](#noop)
+- [nor](#nor)
+- [not](#not)
+- [nth](#nth)
+- [or](#or)
+- [pick](#pick)
+- [pipe](#pipe)
+- [pipeAll](#pipeall)
+- [plus](#plus)
+- [product](#product)
+- [property](#property)
+- [push](#push)
+- [put](#put)
+- [reduce](#reduce)
+- [reduceRight](#reduceright)
+- [replace](#replace)
+- [shallowClone](#shallowclone)
+- [shave](#shave)
+- [signum](#signum)
+- [some](#some)
+- [split](#split)
+- [startsWith](#startswith)
+- [startsWithAt](#startswithat)
+- [sum](#sum)
+- [tail](#tail)
+- [take](#take)
+- [takeUntil](#takeuntil)
+- [takeWhile](#takewhile)
+- [test](#test)
+- [times](#times)
+- [uncurry](#uncurry)
+- [uncurry3](#uncurry3)
+- [values](#values)
+- [xor](#xor)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
whoops somehow the toc got duplicated.
can you clean that? 
1. remove the toc (not the doctoc comment!)
2. run `npm run toc`